packages/cli-utils/src/commands/turbo/thread.ts

```diff
@@ -85,6 +94,12 @@ async function* _runTurbo(params: TurboParams): AsyncIterableIterator<TurboSigna
       warnings,
     };
   }
+
+  await rm(overrideFile.compilerNode.fileName);
```

Comment:
If we're actually creating these files, in theory, just as `__gql-tada-override__.d.ts` above, we should aim to create these virtually 🤔

It'd likely be good to abstract this part anyway into a small helper.

Generally the approach of:
- create `ts-morph` project with `skipAddingFilesFromTsConfig: true`
- adding virtual files
- `addSourceFilesFromTsConfig`
Should be abstractable in theory and allow us to reuse this across all commands without forgetting it